polylogue/drive.py

```
@@ -106,31 +107,41 @@ def _retry(
+    def _before(retry_state) -> None:
+        nonlocal attempts
+        attempts = retry_state.attempt_number
+
+    def _after(retry_state) -> None:
+        if notifier and retry_state.outcome.failed:
+            exc = retry_state.outcome.exception()
+            delay = base_delay * (2 ** (attempts - 1))
+            try:
+                notifier(operation=operation, attempt=attempts, total=max_attempts, error=exc, delay=delay)
+            except Exception:
+                pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)